en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,267 @@
+do the same for stylesheets. So the things you have to do is to tell grunt 
+when to lauch these tasks and how to add the two 'functionnality' files to the final javascript and
+stylesheet files.
+
+To add the media javascript task, modify the main javascript task (``grunt_task/javascript_task.js``):
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        grunt.registerTask('javascript',
+            [
+                'coffee:discovering',
+                'coffee',
+                'concat:smartadminjs',
+                'concat:libjs',
+                'concat:orchestrajs',
+                'concat:mediajs',
+                'concat:js'
+            ]
+        );
+    };
+
+When the 'javascript' task will be run, the ``concat:mediajs`` task will be run.
+
+You can do the same for the media stylesheet by modifying the main css task (``grunt_task/css_task.js``):
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        grunt.registerTask('css',
+            [
+                'less:discovering',
+                'less',
+                'concat:smartadmincss',
+                'concat:libcss',
+                'concat:orchestracss',
+                'concat:mediacss',
+                'concat:css',
+                'cssmin'
+            ]
+        );
+    };
+
+To include the result of the first pass to the final javascript file, alter the
+``grunt_tasks/options/concat.js.js`` file:
+
+.. code-block:: javascript
+
+    module.exports = {
+        src: [
+            'web/built/smartadmin.js',
+            'web/built/lib.js',
+            'web/built/orchestra.js',
+            'web/built/media.js'
+        ],
+        dest: 'web/js/all.js'
+    };
+
+That way, the final javascript file (``all.js``) will include the smartadmin functionnality, followed
+by the lib functionnality, and so on...
+
+You can do the same thing with stylesheets by modifying the ``grunt_tasks/options/concat.css.js``
+file:
+
+.. code-block:: javascript
+
+    module.exports = {
+        src: [
+            'web/built/smartadmin.css',
+            'web/built/lib.css',
+            'web/built/orchestra.css',
+            'web/built/media.css'
+        ],
+        dest: 'web/css/all.css'
+    };
+
+At that point, all is connected, you can run the grunt command (``./node_modules/.bin/grunt``) to
+regenerate the ``all.js`` and ``all.css`` files, which will now include the ``open-orchestra-media-admin-bundle``
+assets.
+
+
+.. _add your specific tasks / options to your bundles:
+
+Adding your specific tasks / options to your bundles
+----------------------------------------------------
+
+The last thing to see in this documentation is how to create your specific tasks for your own
+bundles. In the following section we will only talk about javascript files, but the process is the
+same for the stylesheet files. Assume our bundle is named foo.
```

Comment:
Assume ... should be in a new line : Let's assume you have created the `FooBundle`